templates/bot/js/notification-function-base/README.md

```diff
@@ -27,7 +27,7 @@ If you selected `http` trigger, you can test it:
 >
 > To run locally, you will need:
 >
-> - `Node.js` installed locally (recommended version: 14)
+> - `Node.js` installed locally (recommended version: 16)
```

Comment:
Some readme changes already on main branch (https://github.com/OfficeDev/TeamsFx/pull/6349)
Do you need to wait for main merge back to dev to avoid conflict?